.pre-commit-config.yaml

```diff
@@ -56,3 +56,12 @@ repos:
         args:
           - --fix=lf
       - id: trailing-whitespace
+  - repo: https://github.com/python-jsonschema/check-jsonschema
+    rev: cb934cfd6d2eea1aed7b20e4e9e6339d5952322c # 0.28.0
```

Comment:
I assume `renovate` will use this syntax in the future. How does this play with `pre-commit autoupdate`?

Reply:
😢 `pre-commit autoupdate` doesn't like it, and just changes it back to a version number. So I've switched the SHAs to version numbers.

Reply:
Yeah, I thought that might be the case. The `pre-commit` maintainer is pretty stubborn, too, so he likely won't change the format.